CMakeLists.txt

```diff
@@ -17,8 +17,8 @@ if(${DT_VERSION} MATCHES "^([0-9]+)\\.([0-9]+)\\.([0-9]+)(-(.*))?$")
 else()
     message(FATAL_ERROR "Failed to parse version string: ${DT_VERSION}")
 endif()
-configure_file(${CMAKE_SOURCE_DIR}/src/version.h.in ${CMAKE_BINARY_DIR}/src/version.h @ONLY)
-configure_file(${CMAKE_SOURCE_DIR}/lnp/manifest.json.in ${CMAKE_BINARY_DIR}/lnp/manifest.json @ONLY)
+configure_file(./src/version.h.in ${CMAKE_BINARY_DIR}/src/version.h @ONLY)
```

Comment:
Similar to what you did in DFHack, I think you should use `${projectname_SOURCE_DIR}`.  The project name above is `DwarfTherapist`, so `${DwarfTherapist_SOURCE_DIR}` should work.